test/image/mocks/z-new_tickmode_sync.json

```diff
@@ -0,0 +1,78 @@
+{
+   "data": [
+      {
+         "name": "Apples",
+         "type": "bar",
+         "x": [
+            "Jan",
+            "Feb",
+            "Mar",
+            "Apr",
+            "May"
+         ],
+         "y": [
+            232,
+            2506,
+            470,
+            1864,
+            -190
+         ]
+      },
+      {
+         "name": "Oranges",
+         "type": "scatter",
+         "x": [
+            "Jan",
+            "Feb",
+            "Mar",
+            "Apr",
+            "May"
+         ],
+         "y": [
+            -0.8,
+            0.09,
+            0.01,
+            0.13,
+            0.42
+         ],
+         "yaxis": "y2"
+      }
+   ],
+   "layout": {
+      "margin": {
+         "t": 40,
+         "r": 70,
+         "b": 40,
+         "l": 70
+      },
+      "width": 700,
+      "legend": {
+         "orientation": "h",
+         "x": 0.6,
+         "y": 1.1
+      },
+      "yaxis": {
```

Comment:
If I try to `minor` ticks to both `yaxis` and `yaxis2` e.g. by adding 
```json
"minor": { "showgrid": true },
```
to the axes objects, I get strange girds at the moment.
Could you please fix that?

Reply:
Yep, fixed.